src/main/kotlin/task11/ktor/task01/taskD/taskD.kt

```diff
@@ -0,0 +1,28 @@
+package task11.ktor.task01.taskD
+
+import io.ktor.locations.Location
+import io.ktor.routing.routing
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import task11.ktor.task01.setupApplication
+
+// TODO - Create a fake database and populate it
+// Hint: Use something simple to represent the DB - for example a mutable collection
+
+
+@Location("/api")
+class Root {
+
+    @Location("/cocktails")
+    class Cocktails
+}
+
+fun main() {
+    embeddedServer(Netty, 8089) {
+        setupApplication()
+
+        routing {
+            // TODO - Handle POST requests and update your fake database
```

Comment:
Kanskje lage en tom routing for `put` så det er tydelig at det er snakk om å lage en put/post request for å oppdatere collection.